content/hi/security-chaos-engineering.md

```diff
@@ -0,0 +1,39 @@
+---
+title: सुरक्षा अव्यवस्था इंजीनियरिंग  (Security Chaos Engineering)
+status: Completed
+category: अवधारणा
+tags: ["security", "methodology", ""]
+---
+
+## यह क्या है
+
+
+सुरक्षा अव्यवस्था इंजीनियरिंग या एससीई एक ऐसी शाखा है जो की [सीमावर्ती प्रणाली](/chaos-engineering/) पर आधारित है।
+एससीई वितरित प्रणाली पर सक्रिय सुरक्षा प्रयोग करता है जिससे प्रणाली की क्षमता में विश्वास बढ़ता है ताकि यह
+किसी भी तरह की उपेक्षा या बुरी शर्तों के साथ भी संचालित किया जा सके। सुरक्षा अव्यवस्था इंजीनियर साइंटिफिक मेथड लूप का उपयोग करते 
+हुए इसकी प्रयोगशीलता को बढ़ाते हैं, जिसमें स्थिरावस्था, अनुमान, निरंतर सत्यापन,
+सबक सीखना और कम से कम संभव नुकसान के उपचार के लागू होने के लिए शामिल होते हैं।
+
+
+## समस्या
+
+
+[साइट अवारी इंजीनियर](/site-reliability-engineering/)(SREs) और साइबर सुरक्षा इंजीनियरों के लिए मुख्य प्राथमिकता होती है कि वे सेवा
+को त्वरित रूप से पुनर्स्थापित करें जिससे जीरो डाउनटाइम और व्यवसाय पर असर को कम करने का लक्ष्य होता है।
+SREs और साइबर सुरक्षा इंजीनियर दोनों पूर्व-विफलता और उसके बाद के घटनाओं से निपटते हैं।
+अधिकांश सुरक्षा मुद्दे जल्दी से खोजने और ठीक करने में चुनौतीपूर्ण होते हैं, जो एप्लिकेशन या सिस्टम के कार्यक्षमता पर असर डालते हैं।
+साथ ही, सुरक्षा घटनाएं विकास चरण के दौरान खोजना आमतौर पर मुश्किल होता है।
+
+
+## समाधान
+
+सुरक्षा अव्यवस्था इंजीनियरिंग [अवलोकनीयता](/observability/) और साइबर प्रतिरोध क्षमता अभ्यासों के आधार पर बनाई गई है।
+इसका उद्देश्य "अज्ञात अज्ञात" को खोलना है और सिस्टम में आत्मविश्वास बढ़ाना है,
+साइबर प्रतिरोध क्षमता बढ़ाकर और अवलोकनीयता को सुधारकर।
+
+इंजीनियरिंग टीम धीरे-धीरे सुरक्षा संबंधी चिंताओं के लिए जागरूकता में सुधार करेंगे।
+जटिल बुनियादी संरचना, प्लेटफॉर्म और वितरित सिस्टमों के अंदर।
```

Comment:
```suggestion
इंजीनियरिंग टीम धीरे-धीरे सुरक्षा संबंधी चिंताओं के लिए जागरूकता में सुधार करेंगे
जटिल बुनियादी संरचना, प्लेटफॉर्म और वितरित सिस्टमों के अंदर।
```

Reply:
Kindly Update all the changes as suggested by @bishal7679